helix-term/src/keymap.rs

```diff
@@ -184,7 +184,7 @@ impl<'de> serde::de::Visitor<'de> for KeyTrieVisitor {
         S: serde::de::SeqAccess<'de>,
     {
         let mut commands = Vec::new();
-        while let Some(command) = seq.next_element::<&str>()? {
+        while let Some(command) = seq.next_element::<Cow<str>>()? {
```

Comment:
This single line is the gist of the fix. If the rest of the commit is unacceptable, feel free to remove everything but this line